src/libraries/System.Windows.Extensions/src/System/Security/Cryptography/X509Certificates/X509Utils.cs

```diff
@@ -20,7 +20,7 @@ internal static SafeCertContextHandle DuplicateCertificateContext(X509Certificat
 
         internal static SafeCertStoreHandle ExportToMemoryStore(X509Certificate2Collection collection)
         {
-            SafeCertStoreHandle safeCertStoreHandle = SafeCertStoreHandle.InvalidHandle;
+            SafeCertStoreHandle safeCertStoreHandle;
```

Comment:
This can just be declared on the subsequent line where it's initialized.

Reply:
Done